packages/core/src/commons/file.ts

```diff
@@ -0,0 +1,19 @@
+export function getMimeTypeFromExtension(extension: string): string {
```

Comment:
export not necessary
```suggestion
function getMimeTypeFromExtension(extension: string): string {
```